server/api.go

```diff
@@ -529,6 +529,34 @@ func (p *Plugin) searchIssues(w http.ResponseWriter, r *http.Request) {
 	w.Write(resp)
 }
 
+func getPermaLink(siteUrl string, postId string, currentTeam string) string {
+	return fmt.Sprintf("%v/%v/pl/%v", siteUrl, currentTeam, postId)
+}
+
+func getFailReason(code int, repo string, username string) string {
```

Comment:
Thanks for being thorough and finding the edge cases on this @hector2

Reply:
No problem Aaron :D